GPUOffload/GPUOffload.cc

```diff
@@ -1,18 +1,101 @@
 #include "GPUOffload.hh"
 
+#include <memory>
+
 #include "celeritas/CeleritasOffload.hh"
 
+#include "G4Exception.hh"
```

Comment:
Should these be `<G4Foo.hh>` since they're from an upstream dependency? I also just realized that `"celeritas/Types.hh"` may be different than `<celeritas/Types.hh>` since the Celeritas repo also has a `celeritas/` directory